src/store/zap/mutations.js

```diff
@@ -488,23 +488,22 @@ export function setDomainFilter(state, filterEnabledClusterPair) {
   let filter = filterEnabledClusterPair.filter
   state.clusterManager.filter = filter
   state.domains.map((domainName) => {
+    const openDomainValue =
+      state.clusterManager.filterString === ''
+        ? filter.domainFilterFn(domainName, state.clusterManager.openDomains, {
+            enabledClusters: filterEnabledClusterPair.enabledClusters,
+          })
+        : true
```

Comment:
checking the existence of the filter string here instead , if its not missing then open the clusters just as it before